app/initialization.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+from customers.customer import Customer
+from customers.car import Car
+from app.shop import Shop
+
+
+def customer_init(data: json) -> list:
+    customers = []
+
+    for customer in data.get("customers"):
+        car = customer["car"]
+        car_name = Car(car["brand"], car["fuel_consumption"])
+        person = Customer(
+            customer["name"],
+            customer["product_cart"],
+            customer["location"],
+            customer["money"],
+            car_name
+        )
+        customers.append(person)
+    return customers

```

Comment:
Use list comprehension and unpacking (using `**`)
Hint: You can place Car creation inside Customer init

Reply:
Perfect)